### PR TITLE
fix: Fix TS issue with dynamic flow and a CameraCapture bug

### PR DESCRIPTION
**Pull request by richardbata**
## 🖼 Context
This PR had to introduce some `@ts-ignore`s and this [PR](https://github.com/transferwise/balance-flows-web/pull/1234/files) is adjusting the type contract to allow us to remove those ignores.

Also, there is a bug in FireFox with the cameraCapture component used in IDWL verification that is easily fixed

## 🚀 Changes

- Add types to Field
- Fix FireFox bug

## 🤔 Considerations
Working on FireFox
image

## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/dynamic-flows/src/step/cameraStep/cameraCapture/tracking/properties.js [hunk 1]
```diff
@@ -18,11 +18,11 @@ export const getCameraPermissionDeniedTrackingProperties = (error) => ({
...
 
-  const hardwareWidth = videoTrack?.getCapabilities()?.width?.max;
-  const hardwareHeight = videoTrack?.getCapabilities()?.height?.max;
+  const hardwareWidth = videoTrack?.getCapabilities?.()?.width?.max;
+  const hardwareHeight = videoTrack?.getCapabilities?.()?.height?.max;
 
-  const feedWidth = videoTrack?.getSettings()?.width;
-  const feedHeight = videoTrack?.getSettings()?.height;
+  const feedWidth = videoTrack?.getSettings?.()?.width;
+  const feedHeight = videoTrack?.getSettings?.()?.height;
```
Comment (amol-tw) [inline packages/dynamic-flows/src/step/cameraStep/cameraCapture/tracking/properties.js:25]: How does the resolution look like when these properties are `null`? Can you post a screenshot for firefox?

Comment (richardbata) [reply packages/dynamic-flows/src/step/cameraStep/cameraCapture/tracking/properties.js:25]: These values here are actually only used for metrics and not the behaviour of the camera

Comment (richardbata) [reply packages/dynamic-flows/src/step/cameraStep/cameraCapture/tracking/properties.js:25]: Added a screenshot

Comment (amol-tw) [reply packages/dynamic-flows/src/step/cameraStep/cameraCapture/tracking/properties.js:25]: 🚀🚀🚀

**Review by amol-tw — APPROVED** [review]

**Pull request merged**